lib/rex/proto/kerberos/helper.rb

```diff
@@ -0,0 +1,87 @@
+module Rex::Proto::Kerberos
+
+  class Helper
```

Comment:
It looks like a lot of these 'helpers' are only used once, and don't appear to add much extra semantic clarity in most cases - should we nuke this file and inline the implementations? 👀